extensions/podman/package.json

```diff
@@ -162,29 +162,29 @@
       "title": "Podman Setup",
       "steps": [
         {
-          "id": "checkPodmanInstalled",
+          "id": "checkInstalledCommand",
           "title": "Checking for Podman installation",
-          "command": "podman.onboarding.checkPodmanInstalled",
+          "command": "podman.onboarding.checkInstalledCommand",
           "completionEvents": [
-            "onCommand:podman.onboarding.checkPodmanInstalled"
+            "onCommand:podman.onboarding.checkInstalledCommand"
           ]
         },
         {
-          "id": "startPodmanInstallation",
+          "id": "welcomeInstallationView",
           "title": "We could not find Podman. Let's install it!",
           "when": "onboardingContext:podmanIsNotInstalled"
         },
         {
-          "id": "checkPodmanRequirements",
+          "id": "checkRequirementsCommand",
```

Comment:
Shouldn't end with View like following one ?

Reply:
This step executing a command (even if there is a title and some description), it shoudl be considered as a command.